composer.json

```diff
@@ -39,7 +39,6 @@
         "pulse00/ffmpeg-bundle": "^0.6",
         "oro/doctrine-extensions": "^1.0",
         "goodby/csv": "^1.3",
-        "doctrine/data-fixtures": "1.1.*",
```

Comment:
Don't we need that anymore?

Reply:
@danrot comes with the doctrine/doctrine-fixtures-bundle and needs a higher versions. It seems we did set it to 1.1.* because 1.2.* did drop the php 5.5. support there where we where still no compatible to 7.0